airflow/providers/mysql/hooks/mysql.py

```diff
@@ -61,6 +61,8 @@ def __init__(self, *args, **kwargs) -> None:
 
     def set_autocommit(self, conn: MySQLConnectionTypes, autocommit: bool) -> None:
         """
+        Mysqldb client uses an `autocommit` method rather than an `autocommit` property.
+
         The MySQLdb (mysqlclient) client uses an `autocommit` method rather
```

Comment:
MySQLdb is the old name (Python 2 branch) of mysqlclient, the Python library to connect to MySQL. Since we don’t support Python 2 anymore, this should only mention mysqlclient.